interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
+
+.directive('statusBar', function() {
+  return {
+    restrict: 'E',
+    scope: {
+      msgText: '='
```

Comment:
If you don't have time now, don't worry about the option of `component()` function. I am reading Angular's official doc recently and feel that it is an interesting feature to share with you guys.